tests/test_environment_indexentries.py

```diff
@@ -126,3 +126,21 @@ def test_create_index_by_key(app):
     assert index[0] == ('D', [('docutils', [[('main', '#term-docutils')], [], None])])
     assert index[1] == ('P', [('Python', [[('main', '#term-python')], [], None])])
     assert index[2] == ('ス', [('スフィンクス', [[('main', '#term-0')], [], 'ス'])])
+
+
+@pytest.mark.sphinx('dummy', freshenv=True)
+def test_create_index_with_name(app):
```

Comment:
Could you move this above `test_create_index_by_key()`? It would be better to place testcases for index directive nearly.

Reply:
Done